### PR TITLE
Loader

### PR DESCRIPTION
**Pull request by thabti**
#17 Loader
Screenshot:
screen shot 2017-03-07 at 11 40 29 pm

Wraps around a container, needs to pass a boolean. I can imagine this uses a redux state to determine if the loader is active or not. 

Usage:

```js
@contect(...)
class AppContainer extends Component {


 render() {
  return (
<Loader isActive={this.props.isSuccessfulHttpCall}>
  <App {...props} />
</Loader>
}

}

### CODE REVIEW

File: .storybook/config.js [hunk 1]
```diff
@@ -15,7 +15,7 @@ setOptions({
   url: 'https://github.com/quran/common-components',
   goFullScreen: false,
   showLeftPanel: true,
-  showDownPanel: false,
+  showDownPanel: true,
```
Comment (thabti) [inline .storybook/config.js:18]: so we can see onClick actions logs

Comment (mmahalwy) [reply .storybook/config.js:18]: +1

File: src/Loader/style.scss [hunk 2]
```diff
@@ -0,0 +1,60 @@
+$brand-teal-color: #2CA4AB;
```
Comment (mmahalwy) [inline src/Loader/style.scss:1]: hmm, I think our indentation is not the same :(. This is 4 spaces/1 tab?

**Review by mmahalwy — APPROVED** [review]

File: src/Loader/style.scss (hunk 2 above)
Comment (thabti) [reply src/Loader/style.scss:1]: can we add an eslint for that?

Comment (thabti) [reply src/Loader/style.scss:1]: wait silly me this is scss, what is the indentation we following?

Comment (mmahalwy) [reply src/Loader/style.scss:1]: Two spaces ;) https://github.com/quran/common-components/blob/master/src/Navbar/style.scss

Let's add: https://github.com/stylelint/stylelint

Comment (thabti) [reply src/Loader/style.scss:1]: @mmahalwy we need to have a CONTRIBUTING.md

Comment (thabti) [reply src/Loader/style.scss:1]: updated to 2 spaces.

**Pull request merged**